<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefRangeSetupDialog</class>
 <widget class="QDialog" name="RefRangeSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RefRangeSetupDialog</string>
  </property>
  <widget class="QLabel" name="lbPlt">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PLT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMch">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MCH</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRdwSd">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RDW-SD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPct">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PCT</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LYM%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMcv">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MCV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHct">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>HCT</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlcr">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PLCR</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRbc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RBC</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPdw">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PDW</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbGroup">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbWbc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>WBC</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRdwCv">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RDW-CV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMchc">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MCHC</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlcc">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PLCC</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMpv">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MPV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHgb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>HGB</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWbcLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWbcHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>10.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRbcLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3.5</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRbcHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>5.5</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leLymPerLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leLymPerHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHgbHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>160.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHgbLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>110.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMidPerHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>9.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MID%</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMidPerLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leLymNumLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0.8</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leGranPerHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>70.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>GRAN%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LYM#</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leLymNumHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leGranPerLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>50.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMidNumLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>GRAN#</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leGranNumLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMidNumHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0.9</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLymPer_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MID#</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leGranNumHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>7.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePltHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHctLow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>37.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMchLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>27.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRdwCvLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMchcHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMcvHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>95.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRdwSdHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMchcLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMchHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>31.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRdwCvHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePltLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRdwSdLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMcvLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>82.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHctHigh">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>50.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePdwLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMpvHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePlcrLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePctLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePlcrHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMpvLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePctHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePdwHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePlccHigh">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>40.0</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePlccLow">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>20.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSpecies">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Species</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSpecies">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DigitLineEdit</class>
   <extends>LineEdit</extends>
   <header>digitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
